blog/27-03-2023-feature-flags-if-statements.md

```diff
@@ -0,0 +1,64 @@
+---
+slug: 'feature-flags-if-statements'
+title: 'Feature Flags are just if Statements, Right?'
+date: 2023-03-27
+authors: [agardnerit]
+description: 'Feature Flags are Just if Statements, Right?'
+keywords: [feature flags, openfeature]
+tags: [feature flags]
+draft: false
+---
+
+Most feature flag explainers begin by explaining that feature flags are glorified if statements.
```

Comment:
I agree with the idea that this is the naive view, but I'm not sure most blog posts or explainers take that view. I would consider rephrasing this to something that suggests that `feature flags == env vars` is the default naive or primitive implementation.